lib/sift/client.rb

```diff
@@ -34,7 +42,13 @@ def initialize(http_response, http_response_code)
     # == Returns:
     #   true on success; false otherwise
     def ok?
-      0 == @api_status.to_i
+      #if there is no content expected, use HTTP code
+      if Response::HTTP_CODES_WITH_NO_BODY.include? @http_status_code
+        204 == @http_status_code
+      # otherwise use API status
```

Comment:
http://danknox.github.io/2013/01/27/using-rubys-native-nethttp-library/  Link to my understanding of the Net constants.  This would be helpful for figuring out if a response should have a body (i.e. http codes 204 and 304 should both be of the "class" Net::HTTPNoContent), however the value of Net::HTTPNoContent is not necessarily 204 or 304, so we'd be adding an additional validation variable.